### DISCUSSION TITLE
Using `wsh` in Powershell Core (PS7)

### DISCUSSION THREAD

**Discussion by Printoiid** [wavetermdev/waveterm#1472] [General]
### Feature description

Right now, the `wsh` command only works in PS5 on Windows, it would be great if this was included in PS7 as well since a seemingly larger majority are using PS7

### Implementation Suggestion

Add the command to the default PS7 path when installing Wave Terminal

### Anything else?

n/a

**Comment by esimkowitz** [comment]
Ah are you opening pwsh from PS5? We don't support adding wsh to path for subshells at the moment. We have an FAQ for setting Git Bash as the default local terminal, you can follow these same instructions for adding pwsh https://docs.waveterm.dev/faq#on-windows-how-can-i-use-git-bash-as-my-default-shell

If you do this, wsh will be loaded
